src/Fixer/Basic/BracesFixer.php

```diff
@@ -354,13 +354,29 @@ function ($item) {
                     continue;
                 }
 
-                if (1 === $nestLevel && $nestToken->equalsAny([';', '}'])) {
+                if (1 === $nestLevel && $nestToken->equalsAny([';', '}', [T_COMMENT]])) {
                     $nextNonWhitespaceNestIndex = $tokens->getNextNonWhitespace($nestIndex);
                     $nextNonWhitespaceNestToken = $tokens[$nextNonWhitespaceNestIndex];
+                    $prevNonWhitespaceNestIndex = $tokens->getPrevNonWhitespace($nestIndex);
+                    $prevNonWhitespaceNestToken = isset($tokens[$prevNonWhitespaceNestIndex]) ? $tokens[$prevNonWhitespaceNestIndex] : null;
```

Comment:
`isset` can be placed with `null === prevNonWhitespaceNestIndex` , as the returned value is either a valid index (`int`) for the Tokens or `null`